apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,390 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { z } from 'zod';
+import { assetPath, compiledScriptTag, nodeModulesAssetPath } from '../../lib/assets';
+import { AssessmentQuestionSchema, IdSchema, TopicSchema } from '../../lib/db-types';
+import { Modal } from '../../components/Modal.html';
+
+export const AssessmentQuestionRowSchema = AssessmentQuestionSchema.extend({
```

Comment:
One thing I'd like for us to consider here: when a column is just directly selected from another table, we could use that table's schema as the source of the column schema. You can see an example of that over in https://github.com/PrairieLearn/PrairieLearn/pull/9705/files:

https://github.com/PrairieLearn/PrairieLearn/blob/4ac0a307fd0f666572fc40c9df95a51e9f83eef6/apps/prairielearn/src/pages/instructorQuestionStatistics/instructorQuestionStatistics.html.ts#L16-L27

@mylesw27 what do you think about this?

Reply:
I didn't know we could do this! I think it makes sense to do this where we can. I started working on updating this `AssessmentQuestionRowSchema` locally but would like to see #9705 land so I can use the `alternativeGroupSchema` where applicable.

Reply:
Sounds good! I landed that PR, so you should be able to pick up that new schema now.

Reply:
Great! Updated the `AssessmentQuestionRowSchema` in [4a46164](https://github.com/PrairieLearn/PrairieLearn/pull/9680/commits/4a46164848ad6ee5192bdb5ab1981026354e6007) to use source table schemas for column schemas.